Code_parser.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+#include <fstream>
+#include <string>
+#include <regex>
+#include <stack>
+
+namespace fs = std::filesystem;
+using namespace std;
+
+class Code_parser
+{
+	private:
+		int blank_lines_count
```

Comment:
It is better to declare the variable and then initializing it.

Reply:
Resolved